### DISCUSSION TITLE
feature request: fallback image for ui.image() and ui.interactive_image() for when URL/source unavailable

### DISCUSSION THREAD

**Discussion by h0uter** [zauberzeug/nicegui#2688] [Ideas / Feature Requests]
I can use the following code to add a live stream element with a fallback image. This is useful because in some testing scenario's when the live stream is unavailable.

```py
from nicegui import ui

livestream_url = f"http://{LIVESTREAM_HOST_IP_ADDRESS}:{LIVESTREAM_PORT}"

livestream_with_fallback_image_html = f"""
<object data="{livestream_url}" type="image/png">
	<img src="http:///i.imgur.com/hfM1J8s.png" alt="Fallback image if livestream is offline.">
</object>"""

ui.html(livestream_with_fallback_image_html)

ui.run()
```

However I would like to use the `ui.interactive_image()` so I can mark things on the live stream using mouse clicks. What is the recommended way to include a fallback image in a similar way with the `ui.interactive_image()` element?

**Comment by rodja** [comment]
I see the benefits of this, yes. Anyone interested in taking this on?
